### ISSUE TITLE
Allow multiple source code files per submission

### ISSUE THREAD

**Issue by moschlar** [moschlar/SAUCE#94]
Don't know why this hasn't existed as an issue...
Clearly the biggest issue at the moment!

**Comment by Ayutac** [comment]
I agree, important issue, especially when different classes for an exercise are submitted.

**Comment by moschlar** [comment]
Dear people, please learn about the languages you are teaching.

Even with Java, you can easily have more than one class definition in **one** java file. The  restriction is that only **one** of those classes may be ``public``. So if you just define all other classes without the ``public`` keyword, it works. It just works.

### PoC:

https://sauce.zdv.uni-mainz.de/events/eip-ws14/sheets/8/assignments/802/similarity/diff/13598/14281
![screenshot from 2015-01-13 11 06 12](https://cloud.githubusercontent.com/assets/228095/5719023/36aaae94-9b14-11e4-9e2c-ea72ea4de0e8.png)
![screenshot from 2015-01-13 11 06 56](https://cloud.githubusercontent.com/assets/228095/5719035/4e6fe6ca-9b14-11e4-816c-862d442fcbfc.png)

**Comment by Ayutac** [comment]
Okay, now that's something they could have teached my in university.

**Comment by moschlar** [comment]
I told Kai that, too, when he asked what he could do about multiple classes - dunno why he didn't "implement" it ;)
